### DISCUSSION TITLE
Storing OPENAI_API_KEY

### DISCUSSION THREAD

**Discussion by loranger** [plandex-ai/plandex#190] [Q&A] [answered]
Hello, is there a way to store the OPENAI_API_KEY variable within the project?
I'd like to avoid exporting the key each time I start a new shell session nor to store the key globally for the whole sytem

**Comment by danenania** [comment, marked as answer]
Hi @loranger, there isn't yet. You could use something like [direnv](https://direnv.net/) to avoid the need to export every time.

We're also working on an integrated models mode for Plandex Cloud that will give you the option of buying credits from us rather than bringing your own account/api key.
